playbooks/roles/wordpress/templates/wordpress-nginx.j2

```diff
@@ -0,0 +1,87 @@
+server {
+        listen *:80;
+        server_name {{ LANDING_HOSTNAME }} www.{{ LANDING_HOSTNAME }};
+
+        {{ NGINX_REWRITES }}
+
+        if ($http_x_forwarded_proto = "http") {
+                return 301 https://$server_name$request_uri;
+        }   
+
+        root /var/www/{{ LANDING_HOSTNAME }};
+        index index.html index.php;
+
+        access_log  /var/www/log/{{ LANDING_HOSTNAME }}_nginx_access.log;
+        error_log /var/www/log/{{ LANDING_HOSTNAME }}_nginx_error.log error;
+
+        location ~ [^/]\.php(/|$) {
+                fastcgi_split_path_info ^(.+?\.php)(/.*)$;
+                if (!-f $document_root$fastcgi_script_name) {
+                        return 404;
+                }   
+
+                fastcgi_pass unix:/var/run/php/php5-fpm.sock;
+                fastcgi_index index.php;
+                # include fastcgi_params;
+                include fastcgi.conf;
+        }   
+
+        # deny access to .htaccess files
+        location ~ /\.ht {
+                deny  all;
+        }   
+
+        location /files {
+                autoindex off;
+                autoindex_exact_size off;
+                autoindex_localtime on; 
+
+                #auth_basic "Restricted";
+                #auth_basic_user_file /var/www/.htpasswd_opencraftstaff;
+        }   
+
+        # Wordpress config
```

Comment:
@Agrendalath, can we keep the global include configuration in `global/wordpress.conf` like it is done now so that it is reusable?

Reply:
Sure :+1: